_schema.graphql

```diff
@@ -3266,6 +3267,7 @@ type Fair {
   # A formatted description of the start to end dates
   exhibition_period: String
   has_full_feature: Boolean
+  has_incorrect_snake_case: Boolean
```

Comment:
<!--
  1 failure:  Found addition us...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: _schema.graphql;
  Line: 3270;
-->
- :no_entry_sign: Found addition using snake_case instead of camelCase: has_incorrect_snake_case